### ISSUE TITLE
Resolve issues caused by apostrophe in flower names

### ISSUE THREAD

**Issue by Andreamc56** [kenwebb/gwpEggplant#23]
Tom's feedback:
**Row 30 (Erythronium americanum):** the apostrophe in the French name shows as a non-printing character on the details page. It displays correctly in the pick list of French names, but if you choose that name there are 0 results. There are two other instances of this problem noted below at rows 61 and 129, but apart from those all other occurrences of d' in French names display correctly on the details pages.
**Row 61 (Viburnum lantanoides):** the apostrophe in the French name shows as a non-printing character on the details page. It displays correctly in the pick list of French names, but if you choose that name there are 0 results.
**Row 129 (Tussilago farfara):** the apostrophe in the French name shows as a non-printing character on the details page. It displays correctly in the pick list of French names, but if you choose that name there are 0 results.  
**Row 158 (Dianthus armeria):** the diphthong OE at the beginning of the French name shows as a non-printing character on the details page. It displays correctly in the pick list of French names, but if you choose that name there are 0 results.

**Comment by kenwebb** [comment]
I fixed the apostrophe problems that Tom found. Two separate apostrophe-like characters were used instead of '. I also fixed several sub_family names in the same way. And I fixed the diphthong OE in Dianthus armeria. I made all of these changes to plants.js .

In addition, I replaced four apostrophe-like characters in localizedStrings.js :
-Érythrone d’Amérique,
+Érythrone d'Amérique,
 
-Tussilage pas d’âne,
+Tussilage pas d'âne,
 
-Viorne bois d’orignal,
+Viorne bois d'orignal,
